test/testrunner.py

```diff
@@ -47,8 +47,8 @@ def run_sagemaker_local_tests(images):
     sm_tests_tar_name = "sagemaker_tests.tar.gz"
     run(f"tar -cz --exclude='*.pytest_cache' --exclude='__pycache__' -f {sm_tests_tar_name} {sm_tests_path}")
     ec2_client = boto3.client("ec2", config=Config(retries={"max_attempts": 10}), region_name=DEFAULT_REGION)
-    for image in images:
-        sm_utils.execute_local_tests(image, ec2_client)
+    # for image in images:
+    #     sm_utils.execute_local_tests(image, ec2_client)
```

Comment:
Why is this commented out? Also, this is not the latest version of the sm local code. Please merge from master to get the latest version of this code.

Reply:
It is already green on PR, so reran just SM remote tests.

Reply:
This PR was created base on latest master available at that time, and master is going to change most probably. If we keep same rerunnning tests again for no Dockerfile change, PR will get delayed